src/test/mock/__files/RedirectCollector.page.html

```diff
@@ -10,6 +10,8 @@
 				CollectorModule,
 				RedirectCollector,
 				RestEventWriter,
+				positionResolver,
```

Comment:
A test without that new resolver would have been nice to assert the default is still working.

Reply:
done :+1: